oop/src/views/AccountTypePage.vue

```diff
@@ -71,6 +76,7 @@
                         :set="v = $v.dependentPhns.$each[index]"
                         class='mt-3'>
                       <PhnInput :label='"PHN: Dependent " + (index + 1)'
+                                id='phn'
```

Comment:
Since this is a repeated component, this might produce unwanted labelling (example: the initial 5 `PhnInput` components would have the same `id`). You may want to use `:id="'phn' + index"` instead. Did this pass the Axe accessibility tool? I thought "id"s had to be unique.